app-service-template/res/configuration.yaml

```
@@ -0,0 +1,82 @@
+# TODO: Go here for detailed information on Application Service configuration:
+#       https://docs.edgexfoundry.org/latest/microservices/application/GeneralAppServiceConfig/
+Writable:
+  LogLevel: "INFO"
+
+  InsecureSecrets:
+    # TODO: Remove if not using HTTPS Webserver
+    HTTPS:
+      path: "https"
+      Secrets:
+        cert: ""
+        key: ""
+
+  Telemetry:
+    Metrics: # All service's metric private configuration metrics must be listed here.
+      # TODO: Remove sample custom metric and implement meaningful custom metrics if any needed.
+      # Custom App Service Metrics
+      EventsConvertedToXML: true
+
+Service:
+# TODO: set this port appropriately,
+#       App service use port assignments 597xx with lower range 00-39 reserved for
+#       app-service-configurable profiles/instances, Rules Engine and examples
+  Port: 59740
+  StartupMsg: "app-new-service Application Service has started"
```

This has to be added to all the configs moving forward
 
```suggestion
  Host: localhost
  Port: 59740
  StartupMsg: "app-new-service Application Service has started"
```